File: order-producer/src/main/java/com/sahakav/coffee_shop/ProducerApplication.java
package com.sahakav.coffee_shop;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import org.springframework.kafka.annotation.EnableKafka;

@SpringBootApplication
@EnableJpaRepositories
public class ProducerApplication {

    public static void main(String[] args) {
        SpringApplication.run(ProducerApplication.class, args);

    }
}

File: order-producer/src/main/scala/com/sahakav/coffee_shop/ProducerApplication.scala
package com.sahakav.coffee_shop


import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.data.jpa.repository.config.EnableJpaRepositories



@SpringBootApplication
@EnableJpaRepositories
class ProducerApplication {

}

object Application extends App {

    SpringApplication.run(classOf[ProducerApplication])
}